packages/snap-store-mobx/src/types.ts

```diff
@@ -63,33 +63,36 @@ export type FinderFieldConfig = {
 	levels?: string[];
 };
 
+export type AutocompleteStoreConfigSettings = {
```

Comment:
This was refactored in the templates branch. `singleResult` is the addition here.